babel.config.json

```diff
@@ -64,6 +59,7 @@
             "version": "^7.10.4"
           }
         ],
+        "@babel/plugin-transform-modules-commonjs",
```

Comment:
Why adding `@babel/plugin-transform-modules-commonjs` here?

Reply:
I'm not sure I fully understand why it's needed, but without it I get a warning:
```
    WARNING in ./src/specmap/lib/refs.js 332:9-14
    "export 'fetch' was not found in 'cross-fetch'
     @ ./src/specmap/index.js
     @ ./src/resolver.js
     @ ./src/index.js
     @ multi ./src/index.js
```
and the resulting bundle breaks with `TypeError: Cannot assign to read only property 'exports' of object '#<Object>'`.

`cross-fetch` uses kind of an odd module definition that I don't really understand, it tries to be compatible with both CommonJS and ESM: https://github.com/lquixada/cross-fetch/blob/master/dist/browser-ponyfill.js#L545

Because `cross-fetch` itself does not use ESM imports, `@babel/plugin-transform-modules-commonjs` does not alter its transpilation. However, it does transform all of _our_ source code's imports, including the imports of `cross-fetch`. For some reason, `webpack` had no problem with this import when it wasn't transpiled. But now that we're running Babel over it, `webpack` can't seem to make sense of it anymore as an ESM import, but doesn't have a problem with it as a CommonJS import. So by running the CommonJS transform over our source files we work around this problem.

Reply:
Thanks for explanation.

Reply:
I did some investigation and the answer it documented really well here: https://github.com/webpack/webpack/issues/4039#issuecomment-419284940. I'm suggesting a changes that will fix this problem and don't use `@babel/plugin-transform-modules-commonjs`.

Reply:
Nice find, that does explain it succinctly.